ber.go

```diff
@@ -330,6 +334,9 @@ func readPacket(reader io.Reader) (*Packet, int, error) {
 	}
 
 	// Read definite-length content
+	if length > MaxPacketLengthBytes {
```

Comment:
`if MaxPacketLengthBytes > 0 && int64(length) > MaxPacketLengthBytes`, and doc the variable as "set to 0 for no limit"